docker/Dockerfile

```diff
@@ -0,0 +1,107 @@
+FROM --platform=linux/amd64 node:16.13.0 AS frontend
+
+WORKDIR /app/oppia
+
+RUN apt-get update -y && apt-get upgrade -y \
+    curl \
+    git \
+    npm
+
+# Installing packages from the package.json file
+COPY package.json .
+COPY scripts/linters/custom_eslint_checks ./scripts/linters/custom_eslint_checks
+
+RUN npm install -g yarn@1.22.15 --force
+# mathjs ESOCKETTIMEDOUT error - there are cases reported for mathjs package when timeout happens while
+# downloading the package using `yarn install` command.
+# The solution is to increase the network timeout, so that the package is downloaded successfully.
+RUN yarn install --network-timeout 1000000
+
+
+# Added platform flag here because -- https://stackoverflow.com/questions/71040681/qemu-x86-64-could-not-open-lib64-ld-linux-x86-64-so-2-no-such-file-or-direc
+FROM --platform=linux/amd64 python:3.8.17-slim-buster AS backend
+
+WORKDIR /
+
+ENV OPPIA_IS_DOCKERIZED="true"
```

Comment:
Is this needed? If yes, are we redinfing it in the docker-compose.yml as well? if yes. let's remove it form docker-compose.yml

Reply:
remved from dockerfile